sds_data_manager/stacks/database_stack.py

```diff
@@ -0,0 +1,97 @@
+"""SdpDatabase Stack"""
+# Installed
+from aws_cdk import (
+    Environment,
+    Stack,
+)
+from aws_cdk import (
+    aws_ec2 as ec2,
+)
+from aws_cdk import (
+    aws_rds as rds,
+)
+from constructs import Construct
+
+
+class SdpDatabase(Stack):
+    """Stack for creating database"""
+
+    def __init__(self,
+                 scope: Construct,
+                 construct_id: str,
+                 env: Environment,
+                 vpc: ec2.Vpc,
+                 rds_security_group,
+                 engine_version: rds.PostgresEngineVersion,
+                 instance_size: ec2.InstanceSize,
+                 instance_class: ec2.InstanceClass,
+                 max_allocated_storage: int,
+                 username: str,
+                 secret_name: str,
+                 database_name: str,
+                 **kwargs) -> None:
+        """
+        Parameters
+        ----------
+        scope : Construct
+            The App object in which to create this Stack
+        construct_id : str
+            The ID (name) of the stack
+        env : Environment
+            CDK environment
+        vpc : ec2.Vpc
+            Virtual private cloud
+        engine_version : rds.PostgresEngineVersion
+            Version of postgres database to use
+        instance_size : ec2.InstanceSize
+            Instance size for ec2
+        instance_class : ec2.InstanceClass
+            Instance class for ec2
+        max_allocated_storage : int
+            Upper limit to which RDS can scale the storage in GiB
+        username : str,
+            Database username
+        secret_name : str,
+            Database secret_name for Secrets Manager
+        database_name : str,
+            Database name
+        """
+        super().__init__(scope, construct_id, env=env, **kwargs)
+
+        self.secret_name = "sdp-database-creds"
+        database_name = "imap"
+        username = "imap_user"
```

Comment:
Would it make sense (long term, not in this PR) to have instrument-specific users?